webserver.py

```diff
@@ -63,6 +63,17 @@ def initialize_state(game_state):
     for i in range(2):
         for j in range(2):
             game_state[("plot" + str(i) + "," + str(j))]['locked'] = 0
+
+    total_seed_count = 0
+    total_crops_planted = 0
+    for seed in GAME_CONFIG['seeds']:
+            total_seed_count += game_state['seedCounts'][seed]
+    for i in range(GAME_CONFIG['field_width']):
+        for j in range(GAME_CONFIG['field_height']):
+            if game_state["plot" + str(i) + "," + str(j)]['seedType'] != 0:
+               total_crops_planted += 1
```

Comment:
Using `5` here is a bit dangerous-- it's possible the cheapest seed is set to cost `6`.

Two good alternatives-- first, you can look through GAME_CONFIG to find the cheapest seed and use that as the threshold value.  The other thing you could do is just use the starting-cash value, as you know for sure if they have that much they can get started.